README.md

```diff
@@ -1,4 +1,20 @@
 # 2022-10-21-git_collaboration
 
+## Exercise 1
+
+1. Create a new repository on github
+	- with a README.md + LICENCE + .gitigore
+2. `clone` the repository to your computer
+	- usually somewhere in `~/git`
+3. In the `README.md` file define `git clone` and `git init`
+4. `git add` and `git commit` your changes
+5. `git push` your changes back up to your github repository
+
+
```

Comment:
maybe we should put a section break here to separate the exercise form the notes

Reply:
ok will go do that